MAUI/ImageEditor/getting-started.md

```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+### Image is edited or not.
+
+The [`IsImageEdited`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_IsImageEdited) property is used to determine whether any editing action has been performed on the image.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="IsImageEdited"
+                Clicked="OnIsImageEditedClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void OnIsImageEditedClicked(object sender, EventArgs e)
+{
+    if (this.imageEditor.IsImageEdited)
+    {
+        this.imageEditor.Save();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## ImageEditor inside stack layout

```

Comment:
Add the same in data form

Reply:
Added in data form.
https://github.com/syncfusion-content/maui-docs/pull/1737